db/migrate/20161104150651_add_broken_link_count_to_local_authority.rb

```diff
@@ -0,0 +1,11 @@
+class AddBrokenLinkCountToLocalAuthority < ActiveRecord::Migration[5.0]
+  def up
+    add_column :local_authorities, :broken_link_count, :integer, default: 0
+
+    LocalAuthority.all.map(&:calculate_count_of_broken_links)
```

Comment:
It's exactly the same functionally, but I feel the shorter syntax is a little more intention revealing. It can be a bit confusing the first time you see it though, I agree. Would you rather it were swapped out for the long-hand?